### DISCUSSION TITLE
Jump to the last buffer (vim's C-^)

### DISCUSSION THREAD

**Discussion by sinefabula** [helix-editor/helix#5067] [Q&A] [answered]
Vim has a very useful feature to switch between the current and the previous buffers. I got very used to that feature of vim and had it mapped to `space, space` which is super fast. How do you do the similar action in Helix? Can it be implemented purely with keybindings? I tried pressing `space, b, enter` and `space, b, down, enter` but Helix sorts buffers seemingly in the chronological order and thus the order remains the same every time I open the picker so such combos do not work.

The obvious alternative is to open two files side by side but sometimes (working on a laptop, having something already open in a split) switching is just an easier option. What do you think?

**Comment by ath3** [comment, marked as answer]
I think what you want is: ga

**Reply by sinefabula** [reply]
Thanks! That, and `gm` for last modified file. Totally missed those!

**Comment by tjstankus** [comment]
This may be a new command since this question was answered, but `gp` will goto previous buffer.
